awsDetonationLab.template

```diff
@@ -3049,6 +3021,48 @@
           "Ref": "wazuhUser"
         }
       }
+    },
+    "KinesisToLambda": {
```

Comment:
Is this change really necessary in this PR?

Reply:
I am pretty sure this change is just the lines being moved due to the actual changes

Reply:
You're right. I have checked these changes are present in master as well 😄 .